rtc/encryption.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: Encryption information for the RTC protocol
+title_nav: RTC Encryption help
+description: Useful information for understanding how encryption is used with RTC
+keywords: rtc encrypt decrypt key rotate signature
+---
+
+This documentation is in progress. Please contact us with any suggestions you think should be here.
+
+> **Caution**: The advice on this page may not guarantee a secure connection. If data secrecy is important to you please consult a security professional.
+
+## Encryption keys are mandatory
+
+RTC requires an encryption key so {{site.companyname}} can prove to ourselves (and any interested customers) that the RTC server does not store document content in a format that we are able to read. We are very focussed on privacy and end-to-end encryption for this feature.
+
+## The difference between content encryption and JWT signing
+
+Encryption is a two way process (encryption and decryption) that transforms data into an encrypted form. Modern encryption algorithms produce an output that is indistinguishable from random data without the decryption key. RTC performs symmetric encryption (meaning encryption and decryption use the same key) on all document content, but the key is never sent to the server. As the key only exists on the client endpoints, the term for this is end-to-end encryption.
```

Comment:
```suggestion
Encrypted data is indistinguishable from random data without the decryption key. The RTC plugin performs client-side symmetric encryption (using the same key for encryption and decryption) on the editor content. The encryption key is provided to the user and your server (the client endpoints), and is never sent to the {{site.{{site.cloudname}}}} servers; providing end-to-end encryption.
```

Reply:
again, I'll correct after committing your suggestions - the key isn't provided _to_ the user, technically, or if it is it's provided by the integrating server. Is `{{site.{{site.cloudname}}}}` valid? I've never seen that before 🧐